### DISCUSSION TITLE
블랙잭 미션

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#6] [General]
# 블랙잭 미션

페어 페어는 헤나였다. 헤나와 처음 만나고 나서 약 40분간 아이스 브레이킹 시간을 가졌다. 페어와 정한 규칙 20분마다 교대했다. 모르는 것은 바로바로 물어본다. 페어가 멍 때리면 괜찮냐고 물어본다. 페어가 마음 상할까 봐 하고 싶은 말을 숨기지 …

https://cyma-s.github.io/level1-blackjack/

**Comment by Songusika** [comment]
혹시 도메인 객체의 역할을 부여할 떄 어디까지 이 객체의 책임일지 정하는 팁을까요..?
